.travis.yml

```diff
@@ -41,8 +41,9 @@ script:
   - xcodebuild -showsdks
   - pod --version
   - scripts/prep_all
-  - travis_wait scripts/build_all --verbose
-  - travis_wait scripts/test_all catalog/MDCCatalog.xcworkspace:MDCUnitTests
+  - xcodebuild build test
+#  - travis_wait scripts/build_all --verbose
+#  - travis_wait scripts/test_all catalog/MDCCatalog.xcworkspace:MDCUnitTests
```

Comment:
"xcodebuild build test" does build MDCCatalog, and runs the unit and UI tests.
I'm trying to determine if we can put all of our workspaces in a matrix.
This PR may not turn out to be something we merge.  (Description updated above)